includes/Assets.php

```diff
@@ -34,6 +34,14 @@ public function admin_asset() {
 	        [ 'dashicons' ],
 			WC_CALL_FOR_PRICE_VERSION
         );
+        // Select2 CDN Style Start
+        wp_register_style(
+            'wcp-select2-style',
+            'https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css',
```

Comment:
Kindly download select2 library and save it to asset folder. Load the library from their.